app/vmui/packages/vmui/src/components/Table/TableSettings/TableSettings.tsx

```diff
@@ -105,6 +113,16 @@ const TableSettings: FC<TableSettingsProps> = ({
                 />
               </Tooltip>
             </div>
+            <div
+              className="vm-table-settings-popper-list__item"
```

Comment:
> It would be great to add some spacing to this checkbox to differentiate it from the others, like so:
> 
> ```
> <div className="vm-table-settings-popper-list__item vm-table-settings-popper-list__item_all"...
> 
> &_check-all {
>   padding: 0 0 $padding-global;
>   border-bottom: $border-divider;
> }
> ```

thanks, bro, i have add new style for **check all**.